src/components/MonsterDetailsHead.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function MonsterDetailsHead({ name, type, size, alignment }) {
+  return (
+    <>
+      <Headline>{name}</Headline>
+
+      <SingleInfo>
+        {size} {type}, {alignment}
+      </SingleInfo>
+    </>
+  )
+}
+
+const Headline = styled.h1`
+  color: black;
+  font-family: serif;
+  font-size: 35px;
+  line-height: 35px;
+  text-align: center;
+  width: 100%;
+  margin-bottom: 5px;
+`
+
```

Comment:
Since you don't use empty lines in between styled components in your ActionsAbilities.js and your MonsterDetailsStats.js :
```suggestion
```